src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncExec.scala

```diff
@@ -378,49 +350,141 @@ class HdfsRsyncExec(config: HdfsRsyncConfig) {
         }
     }
 
+    /**
```

Comment:
This function name and the comment here makes me think that ALL files in the source hierarchy will be returned; but this is just returning the files and folders in each dir in srcPathList, right?  

Let's be consistent with the name folders vs. dirs vs directories where we can.  You use dir elsewhere (createDir).  Can we call this maybe just getSrcsChildren  and the next function getDstChildren?  Or maybe listSrcs and listDst?  Or something else?

Reply:
Or maybe, getPathsInSrcs, getPathsInDst?